features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,36 @@
+Given(/^the following users exists$/) do |table|
+  table.hashes.each do |hash|
+    User.create!(hash)
+  end
+end
+
+Given(/^I'm on the "([^"]*)" page$/) do |page|
+  visit root_path
```

Comment:
Why are you not using the `page` variable to decide on which page to visit? 🤔 
Imagine you have another step that says:
```gherkin
Given I am on the "Articles" page
```
What will happen with this, is that it will always visit the `root_path`

What I would do here, is to have some form of branching statement to decide on what page to visit

Reply:
Perhaps not a blocker at this stage, but keep this in mind later on when you start reusing this step definition